docs/src/development/environment-setup.md

```diff
@@ -27,53 +29,65 @@ Please make sure you're using Scarb installed via [asdf](https://asdf-vm.com/) -
 > ```shell
 > $HOME/.asdf/shims/scarb
 > ```
-> 
-> If you previously installed scarb using official installer, you may need to remove this installation or modify your PATH to make sure asdf installed one is always used.
-
 
-> ❗️ **Warning**
-> 
-> If you haven't pushed your branch to the remote yet (you've been working only locally), two tests will fail:
-> 
-> - `e2e::running::init_new_project_test`
-> - `e2e::running::simple_package_with_git_dependency`
+> 💡 **Info**
 > 
-> After pushing the branch to the remote, those tests should pass.
+> If you previously installed Scarb using official installer, you may need to remove that installation or modify your `PATH`
+> to make sure the version installed by asdf is always used.
+
 
 ### Starknet Devnet
-To install it run `./scripts/install_devnet.sh`
+Install it by running `./scripts/install_devnet.sh`
 
-To run `sncast` tests, it is required to set `SEPOLIA_RPC_URL` environment variable, which is
-used to fork the Sepolia testnet network in the devnet tests. This variable can be set in the `.env` file.
 
 ### Universal sierra compiler
 Install the latest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler) version.
 
 
+### Environmental variables
+
+Set `SEPOLIA_RPC_URL` environmental variable to a Sepolia testnet node URL:
+- either manually in your shell
+    ```shell
+    $ export SEPOLIA_RPC_URL="https://example.com/rpc/v0_7" 
+    ```
+- or inside `.env` file (example found in `.env.template` file)
+    ```
+    SEPOLIA_RPC_URL="https://example.com/rpc/v0_7"
+    ```
+
 
 ## Running Tests
-Tests can be run with:
+After performing these steps, you can run tests with:
 
 ```shell
 $ cargo test
 ```
 
+> ❗️ **Warning**
+>
+> If you haven't pushed your branch to the remote yet (you've been working only locally), two tests will fail:
+>
+> - `e2e::running::init_new_project_test`
+> - `e2e::running::simple_package_with_git_dependency`
+>
+> After pushing the branch to the remote, those tests should pass.
 
 ## Formatting and Lints
 
-Starknet Foundry uses [rustfmt](https://github.com/rust-lang/rustfmt) for formatting. You can run the formatter with
+Starknet Foundry uses [rustfmt](https://github.com/rust-lang/rustfmt) for formatting. You can run the formatter with:
 
 ```shell
 $ cargo fmt
 ```
 
-For linting, it uses [clippy](https://github.com/rust-lang/rust-clippy). You can run it with this command:
+For linting, it uses [clippy](https://github.com/rust-lang/rust-clippy). You can run it with:
 
 ```shell
 $ cargo clippy --all-targets --all-features -- --no-deps -W clippy::pedantic -A clippy::missing_errors_doc -A clippy::missing_panics_doc -A clippy::default_trait_access
```

Comment:
nit: This line can be updated to align with the lint alias defined in the [config.toml](https://github.com/foundry-rs/starknet-foundry/blob/master/.cargo/config.toml).
```suggestion
$ cargo clippy --all-targets --all-features -- --no-deps -W clippy::pedantic -A clippy::module_name_repetitions -A clippy::missing_errors_doc -A clippy::missing_panics_doc -A clippy::default_trait_access
```